pol-core/pol/module/uomod.cpp

```diff
@@ -4783,7 +4782,14 @@ bool UOExecutorModule::is_reserved_to_me( Item* item )
 
 BObjectImp* UOExecutorModule::mf_Shutdown()
 {
-  Clib::exit_signalled = true;
+  int exit_code;
+
+  if ( !getParam( 0, exit_code ) )
+  {
+    exit_code = 0;
+  }
```

Comment:
this is the only piece I'm iffy about: what are the scenarios where `getParam()` return false, and what would we want this function to do.
- no parameter at all: shouldn't happen (compiler should always pass an argument)
- passed a different type, say a string: will exit 0 (success).  Is that what we want?